src/rgw/rgw_reshard.cc

```diff
@@ -1108,10 +1048,6 @@ void  RGWReshard::get_logshard_oid(int shard_num, string *logshard)
 
 int RGWReshard::process_all_logshards()
 {
-  if (!store->svc()->zone->can_reshard()) {
-    ldout(store->ctx(), 20) << __func__ << " Resharding is disabled"  << dendl;
-    return 0;
-  }
```

Comment:
don't remove this block, we can't reshard in multisite yet

Reply:
yes, this shouldn't be here.

Reply:
done